manifests/default.pp

```diff
@@ -1,3 +1,11 @@
+file { '/root/.ssh/authorized_keys':
+    ensure => file,
+    group  => 'root',
+    mode   => '755',
```

Comment:
No, you definitely don't want `authorized_keys` to be `755`. First of all, it shouldn't be executable. Then, this is a security file. Downgrade it to `600`.

> Actually, some implementations of SSH won't even allow the dæmon to start without this set of permissions.